attachment_sync_gdrive/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# © 2018 Sunflower IT (http://sunflowerweb.nl)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Attachment Sync Google Drive",
+    "version": "11.0.1.0.0",
+    "category": "Storage",
+    "author": "Sunflower IT,Odoo Community Association (OCA)",
+    "website": "https://sunflowerweb.nl",
+    "license": "AGPL-3",
+    "summary": "Sync Attachments to Google Drive",
+    "depends": [
+        "structured_external_storage_all",
```

Comment:
Why is this dependency needed? Can't users just decide to use `external_file_location` and install the Dropbox module on top of it, without the need for the structured storage module which adds the sync rules?